web/src/message_util.ts

```diff
@@ -112,3 +116,41 @@ export function get_topics_for_message_ids(message_ids: number[]): Map<string, [
     }
     return topics;
 }
+
+export function can_previous_pms_exist(recipient_ids_string: string): {
```

Comment:
"private message" is an old name for "direct messages". we should not use `pms` in a name. I would prefer to just spell out `direct_messages`, or maybe better, something like `get_direct_messages_permission_hints(recipient_ids_string)`.